understanding/22/visible-controls.html

```diff
@@ -40,29 +40,31 @@ <h2>Intent of Visible Controls</h2>
 
       <h3>Cognitive disabilities</h3>
 
       <p>People with low executive function, impaired memory, and other cognitive and learning disabilities may not be able to find hidden controls. If a hidden control is needed in order to progress, this can prevent some users from completing a task. Users who discover controls that display on hover may not remember how to show and operate the controls the next time they interact with the site. So even when a user is able to proceed, time on task can be affected for both one-time and repeated uses.</p>
     
       <h3>Vision loss</h3>
 
-      <p>People with vision loss may not see controls that display only on hover or focus, particularly if they display outside of the viewport. For users who require enlarged or magnified content, the portion of the content in the viewport may be significantly reduced. Low vision users may have 800 x 600 resolution as the default browser zoom. They may look closely at the monitor, say four inches from the display, due to limited acuity (inability to see detail) or Scotomas (blind spots). One of the main challenges for people with vision loss is simply locating related controls on the screen, moving the mouse from control to control, and getting the keyboard to the point where they are looking. They may have a significant amount of head and neck movement while using content. When users don't see something on the screen, they may assume they are missing it — for example, that it’s in their blind spot or hidden in a menu — and go searching until they either find it or give up searching. This success criterion can help reduce that cumulative load.</p>
+      <p>One of the main challenges for people with vision loss is simply locating related controls on the screen, moving the mouse from control to control, and getting the keyboard to the point where they are looking. This Success Criterion can help reduce that cumulative load.</p>
+
+      <p>People with vision loss may not see controls that display only on hover or focus, particularly if they display outside of the viewport. For users who require enlarged or magnified content, the portion of the content in the viewport may be significantly reduced. Low vision users may have 800 x 600 resolution as the default browser zoom. They may look closely at the monitor from a few inches away due to limited acuity (inability to see detail) or Scotomas (blind spots).  They may have a significant amount of head and neck movement while using content. When users don't see something on the screen, they may assume they are missing it — for example, that it’s in their blind spot or hidden in a menu — and go searching until they either find it or give up searching. </p>
```

Comment:
`default browser zoom` probably more aptly `default browser viewport dimension` ? (they may not zoom at all, but have their desktop set to 800x600 instead)